Resources/Public/JavaScript/qucosa.js

```diff
@@ -101,6 +101,11 @@ var validateFormAndSave = function() {
     jQuery("#validDocument").val("0");
     if (validateForm()) {
         jQuery("#validDocument").val("1");
+
+        jQuery("#new-document-form #save").prop("disabled", true);
+
+        jQuery('#new-document-form').submit();
```

Comment:
Why is there now a call to `submit()`? How did this happen before?

Reply:
Because, the browser won't submit the form anymore because the submit button is disabled. You need to do this in javascript IMHO.